config/manifests/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ resources:
 - ../default
 - ../samples
 - ../scorecard
+- ../openshift
```

Comment:
Hey @Josh-Matsuoka, can you remove this line? We can't ship this in our upstream bundles or they will fail to install on Kubernetes.

Reply:
Fixed